src/lib/src/fancy-form/checkbox.component.css

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
The sizes seem inline with Publish's large fancy form fields, but are you planning to override them for the metrics table?